sparta/src/DAG.cpp

```diff
@@ -8,76 +8,98 @@
 
 namespace sparta
 {
-
-    void DAG::CycleException::outPutIssue_(std::ostream & os, bool dot) const
-    {
-        if(dot) {
-            os << "digraph dag_issue {\n";
-            os << "\trankdir=TB;\n";
-            os << "\tnode [shape=record, fontname=Helvetica, fontsize=10];\n\n";
-        }
-        else {
-            os << "DAG CYCLE: " << std::endl;
+    /**
+     * Write a text version of the CycleException's cycle vertex list
+     * @param os
+     */
+    void DAG::CycleException::writeText(std::ostream& os) const {
```

Comment:
Nit: the method name `writeText` is a little vague.  How about something like `outputCycleIssueText`?

Reply:
ok